content/python/concepts/keywords/keywords.md

```diff
@@ -0,0 +1,16 @@
+---
+Title: 'Keywords'
+Description: 'Python keywords are reserved for specific purposes and not available for assignment.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+Certain keywords are reserved in Python and are not available to be assigned as variables or named functions. There are many keywords where some specilize in control-flow (e.g. `if/else` and `for`) while others work as [operators](https://www.codecademy.com/resources/docs/python/operators) (e.g. `and`, `in`, and `is`).
```

Comment:
This is somewhat sparse. I'd maybe include a table with a representative list of reserved keywords with the relevant links.

Reply:
I had a similar idea actually 😄 I would like this to have a `terms` folder with entries for such keywords. This way, a table can be auto-generated upon merge.

Reply:
Maybe term entries for `continue`, `try`, `yield`, etc.